src/apps/chifra/cmd/serve_run.go

```diff
@@ -27,14 +35,25 @@ func validateServeArgs(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-func runServe(cmd *cobra.Command, args []string) {
-	options := ":8080"
-	if len(ServeOpts.port) > 0 {
-		options = " --port " + ServeOpts.port
+func PrintServeSettings() {
+	if ServeOpts.port != ":8080" {
```

Comment:
I presume this is where you fix the bug. Did we have a test for this?

I don't think we did -- in fact, I doubt we had very many tests at all for the server code.

Reply:
It was fixed on line 55, actually. 
I think we don't have a test for it. It seems we only have 2 tests for serve

Reply:
Serve is weird as it has no real purpose from the command line and can't be queries by itself in the server case, so what would there be to test. Plus -- the tests themselves use the server constantly, so I don't think we really need any.